frontend/src/views/Plaza.vue

```diff
@@ -203,7 +212,7 @@
                     variant="primary"
                     class="ml-3 gtag-link-others"
                     @click="onClaimGarrisonXp">
-                    {{$t('plaza.claimXp')}}
+                    {{claimGarrisonXpButtonText}}
```

Comment:
I would prefer to use conditional translations in line, but I think it's very subjective
`isClaimingXp ? $t('plaza.claiming') : $t('plaza.claimXp')`

Notice I would also move "..." to the translation directly^

Example:
![image](https://user-images.githubusercontent.com/34208222/152051365-e91364fd-e7a8-40e0-96ee-cdd48b1a1aa7.png)